### ISSUE TITLE
Importing Tiles On Top Of Each Other

### ISSUE THREAD

**Issue by sonatuzun** [heygleeson/godot-ldtk-importer#12]
Is there currently a way to import 2 tiles that are on the same layer and on top of each other??

My LDtK preview looks like this
![image](https://github.com/heygleeson/godot-ldtk-importer/assets/55955579/d1027866-a100-428b-a523-3af5b9c1ae37)

But the level on Godot looks like this 
![image](https://github.com/heygleeson/godot-ldtk-importer/assets/55955579/15f81c91-a25b-4f82-ae4b-d722a8fa32b9)

You can see the that green moss blocks the tile underneeth

**Comment by sonatuzun** [comment]
It seems that layers are also merged into one layer, why could that be 🤔

**Comment by sonatuzun** [comment]
Verbose output shows that laters are indeed recognized but I don't see them in the final file I wonder why 🤔 
```
=== LEVEL: Level_0 ===
Creating IntGrid Layer: Walls-values
Creating Tile Layer: Walls
Creating IntGrid Layer: Pipes-values
Creating Tile Layer: Pipes
Creating IntGrid Layer: Water-values
Creating Tile Layer: Water

=== LEVEL: Level_1 ===
Creating IntGrid Layer: Walls-values
Creating Tile Layer: Walls
Creating IntGrid Layer: Pipes-values
Creating Tile Layer: Pipes
Creating IntGrid Layer: Water-values
Creating Tile Layer: Water
Built Levels :: 8ms
Resolving 0 entity references...
Built World :: 0ms
Saved Tilesets :: 233ms
Packed World Scene :: 2ms
Saved World Scene :: 16ms
-- LDTK: Finished Import -- (275ms)

```

**Comment by sonatuzun** [comment]
![image](https://github.com/heygleeson/godot-ldtk-importer/assets/55955579/92b58b6a-1af1-4d35-a331-ac44a1e74b5c)
Also different layers of the image are imported correctly (even though there is a problem with opacity :(

**Comment by sonatuzun** [comment]
`var existingTileData := tilemap.get_cell_tile_data(layer_index, cell_grid)
		if not existingTileData:
			tilemap.set_cell(layer_index, cell_grid, tile_source_id, tile_grid, alternative_tile)
		else:
			unplacedTiles.append(tile)
		
	if not unplacedTiles.is_empty():
		create_tile_layer(
		tilemap,
		tile_layer_index,
		layer_data,
		layer_def,
		overflowCount + 1,
		unplacedTiles
		)`

**Comment by sonatuzun** [comment]
There are still some differences but those are mostly due to Godot not handling individual offset's for tiles :( which is a same. If you want to include such solution in the project let me know and I can try to put together a proper pull request

**Comment by sonatuzun** [comment]
Everything works exactly as they should when I turn off random offsets:
![image](https://github.com/heygleeson/godot-ldtk-importer/assets/55955579/06ab212b-8e30-4250-934a-1d14684e420f)
![image](https://github.com/heygleeson/godot-ldtk-importer/assets/55955579/dd1131c7-4969-422e-91ee-fbf49d603a87)

**Comment by sonatuzun** [comment]
If you want to include such solution in the project let me know and I can try to put together a proper pull request

**Comment by heygleeson** [comment]
![Godot_v4 2-stable_mono_win64_24-02-23_11-42](https://github.com/heygleeson/godot-ldtk-importer/assets/1772788/e1b4808d-b5cc-47b1-aafb-7f1c5b2f286a)

There is an import setting "Allow Overlapping Tiles" to allow this behaviour, and should be enabled by default.

**Comment by sonatuzun** [comment]
Oh :D, then I think the AssetLib version is a little bit behind 
![image](https://github.com/heygleeson/godot-ldtk-importer/assets/55955579/7be89f6e-d51b-4917-96ad-458afcb28edf)

**Comment by heygleeson** [comment]
Fixed in [v2.0.1](https://github.com/heygleeson/godot-ldtk-importer/releases/tag/2.0.1)
